modules/packages/FFTW.chpl

```diff
@@ -88,8 +88,12 @@
 
 module FFTW {
 
+  // Define includes and libraries
+  config param FFTW_includes : string ="fftw/fftw3.h, fftw/fftw3_mkl.h";
```

Comment:
I'd expect that the line above would not work and would result in the similar "file not found" error as you reported on the command-line.  I'm guessing that maybe you were experimenting with different values in the source and committed this one instead of one that simply set this to "fftw3.h" (the non-MKL default you said you'd implemented)?